lightning_pose_app/ui/extract_frames.py

```diff
@@ -431,25 +441,50 @@ def _render_streamlit_fn(state: AppState):
         filename = uploaded_file.name.replace(" ", "_")
         filepath = os.path.join(video_dir, filename)
         st_videos.append(filepath)
+
         if not state.run_script:
             # write the content of the file to the path, but not while processing
             with open(filepath, "wb") as f:
                 f.write(bytes_data)
+        
+        # Calculate video duration and frame count
+        cap = cv2.VideoCapture(filepath)
+        fps = cap.get(cv2.CAP_PROP_FPS)
+        frame_count = int(cap.get(cv2.CAP_PROP_FRAME_COUNT))
+        duration = float(frame_count) / float(fps)
+
+        # Create a formatted string with file size, duration, and frame count
+        #file_info = f"{uploaded_file.name} (Video duration: {duration:.2f} seconds, FPS: {fps},Number of frames: {frame_count} frames)"
+
+        # Display the formatted information
+        #st.write(file_info)
+
+        col1.write(uploaded_file.name)
+        col2.write(f"{duration:.2f} seconds")
+        col3.write(str(frame_count))
+
+        # relese video 
+        cap.release()
+    
+    # inseret an empty element to create empty space
+    st.markdown("###")
 
     col0, col1 = st.columns(2, gap="large")
     with col0:
         # select number of frames to label per video
-        n_frames_per_video = st.text_input("Frames to label per video", 20)
+        n_frames_per_video = st.text_input("Frames to label per video", 20,help="Specify the desired number of frames for labelling per video. Our system will then intelligently select these frames to maximize the diversity of animal poses captured within each video, optimizing the training process for your model")
```

Comment:
lines shouldn't exceed 99 characters (you'll see these warnings later when we go over code linting)
for these longer lines, do something like:
```
n_frames_per_video = st.text_input(
    "Frames to label per video", 20,
    help="Specify the desired number ..." \
             "<pick up string here>           ..." \
             "<continue string here>         ..."
```
you can set up VSCode to draw vertical lines at 99 characters to help, I'll show you how to do this tomorrow

Reply:
i won't write this comment for the other strings but make sure to clean the rest up as well

Reply:
string typo: labelling -> labeling

Reply:
Thanks. will do!

Reply:
all amended